massgov/mayflower-patternlab
# DP-8221: Featured Item Link functionality

## Description
Truncates featured item titles past character limit.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-7564)

## Steps to Test
1. in the featured item mosaic component (/?p=organisms-featured-item-mosaic) add a long title to one of the items. You should see that long titles are truncated with ellipses 

## Screenshots
<img width="556" alt="screen shot 2018-03-15 at 6 00 43 pm" src="https://user-images.githubusercontent.com/18662734/37493966-39e20fc2-287d-11e8-9b50-b7125441117a.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application
* Featured Items
* Featured Item Mosaic
